### PR TITLE
TFLite runtime on Linux/macOS/Windows

### PR DESCRIPTION
**Pull request by lissyx**

### CODE REVIEW

File: taskcluster/package.sh [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 set -xe
 
+tflite=$1
```
Comment (reuben) [inline taskcluster/package.sh]: Callers don't pass anything to `package.sh`.

File: taskcluster/tc-cppwin_tflite-ds-tests.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```
Comment (reuben) [inline taskcluster/tc-cppwin_tflite-ds-tests.sh]: This file is identical to `taskcluster/tc-cpp_tflite-ds-tests.sh`, why do we need it?

File: taskcluster/win-build.sh [hunk 3]
```diff
@@ -3,6 +3,11 @@
 set -xe
 
 cuda=$1
+runtime=$1
+
+if [ "${cuda}" != "--cuda" ]; then
+    cuda=
+fi;
```
Comment (reuben) [inline taskcluster/win-build.sh]: Nice hack, but it took me way too much time to understand for what should be a simple script, please separate these into different parameters or make downstream consumers understand `--cuda` vs `--tflite`.

File: taskcluster/win-package.sh [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 set -xe
 
-arm_flavor=$1
+tflite=$1
```
Comment (reuben) [inline taskcluster/win-package.sh]: Callers don't pass anything to `win-package.sh`.

File: taskcluster/tc-cppwin_tflite-ds-tests.sh (hunk 2 above)
Comment (lissyx) [reply taskcluster/tc-cppwin_tflite-ds-tests.sh]: No need

File: taskcluster/win-build.sh (hunk 3 above)
Comment (lissyx) [reply taskcluster/win-build.sh]: Updated downstream to handle properly

**Pull request merged**